standard/lexical-structure.md

```diff
@@ -741,7 +744,192 @@ Each string literal does not necessarily result in a new string instance. When t
 > ```
 > is `True` because the two literals refer to the same string instance. *end example*
 
-#### 7.4.5.7 The null literal
+#### §interpolated-string-literals Interpolated string literals
+
+***Interpolated string literals*** are similar to string literals, but contain ***holes*** delimited by `{` and `}`, wherein expressions can occur. At runtime, the expressions are evaluated with the purpose of having their textual forms interpolated (that is, substituted) into the string at the place where the hole occurs. The syntax and semantics of string interpolation are described in §interpolated-strings).
+
+Like string literals, interpolated string literals can be either regular or verbatim. Interpolated regular string literals are delimited by `$"` and `"`, and interpolated verbatim string literals are delimited by `$@"` and `"`.
+
+Like other literals, lexical analysis of an interpolated string literal initially results in a single token, as per the grammar below. However, before syntactic analysis, the single token of an interpolated string literal is broken into several tokens for the parts of the string enclosing the holes, and the input elements occurring in the holes are lexically analyzed again. This may in turn produce more interpolated string literals to be processed, but, if lexically correct, will eventually lead to a sequence of tokens for syntactic analysis to process.
+
+```antlr
+Interpolated_String_Literal
```

Comment:
I don't know why the nonterminal names are capitalized here but lowercase elsewhere.  They need to be the same.

Reply:
I strongly suspect it's due to the age of the PR - our conventions have changed over time.

Reply:
The ANTLR convention is to use a leading uppercase letter on lexer rule names.